### PR TITLE
added inital html for success-stories (4 stories)

### LINKED ISSUE #16: Our Success Stories Section html of the web page
- [x] Creates div elements
- [x]  Uses h2 element for the title and h4 elements for the names and uses p elements for the paragraph

### LINKED ISSUE #17: the Success-Stories class

### PR DESCRIPTION
**Pull request by inggritenovaleta**
I have added the initial HTML for 'success-stories' section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,7 +53,44 @@ <h1 class="showcase-h1">HELPING NEW TALENT INTO THE TECH WORLD</h1>
...
+    <div class="story_1">
+        <div class="image-container">
+            <img src="public/img/Hanim.jpg" width="150px" alt="Zeynep Hanim">
+        </div>
+        <h4 class="Zeynep">Zeynep Hanim - Optis (Cronos)</h4>
+        <p class="job">Full-stack Developer</p>
+        <p>"HYF is a light that I saw at the bottom as well, a hope that I held onto  as I was building a new life in a new country. It gave me a chance not  only to improve my knowledge but to boost my confidence. Also, it gave me a chance to meet many  wonderful people."</p>
+    </div>  
+    </div>
+    <div class="story_2">
```
Comment (rago89) [inline index.html]: in this div to make more semantic the webpage, would be nice to change the div by an article, also for the other stories, this[ link](https://www.w3schools.com/html/html5_semantic_elements.asp) will help you to understand

**Review by rago89 — COMMENTED** [review]
Hi, Is a Very good structure I'll make some suggestions to make it better

File: index.html (hunk 1 above)
Comment (inggritenovaleta) [reply index.html]: okaay, thank you for the feedback, ill adjust it and ill let you know :)

Comment (inggritenovaleta) [reply index.html]: @rago89  I have made some changes regarding the HTML semantic elements, please have a look, :) thank you in advance

Comment (inggritenovaleta) [reply index.html]: @rago89  I have added the inital css style for success stories and resolve the conflicts. Please kindly have a look :) thank you!

**Comment by rago89** [comment]
Hi @inggritenovaleta  could you please fix the mergin conflicts, try to work with Stephane explanation on last Sunday class

**Review by rago89 — APPROVED** [review]
See no errors

**Pull request merged**